backend/src/pagereq/index.ts

```diff
@@ -69,6 +69,8 @@ function genReturnVal(): PRSReturnVal {
 
 // generate an error'd return value
 function genErrorVal(err: Error): PRSReturnVal {
+  console.log("PAGEREQ ERROR: " + err);
```

Comment:
I know we don't have a real logging system yet, but this should be
```suggestion
  console.error(`PAGEREQ ERROR: ${err}`);
```